CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Pull Requests welcome! We welcome contributions for bugs, as well as additional learning materials to showcase on the website.
+
+## The Five Golden Rules
+
+The simple steps of contributing to any GitHub project are as follows:
+
+1. [Fork the repository](https://github.com/reactjs-id/reactjs.id/fork)
+2. Create your feature branch: `git checkout -b my-new-feature`
```

Comment:
I think we can use the branch prefix with `username`. so,  `git checkout -b username/my-new-feature`.

thanks.

Reply:
good idea!